lib/arborist/build-ideal-tree.js

```diff
@@ -109,7 +111,7 @@ const _peerSetSource = Symbol.for('peerSetSource')
 
 // used by Reify mixin
 const _force = Symbol.for('force')
-const _explicitRequests = Symbol.for('explicitRequests')
+const _explicitRequests = Symbol('explicitRequests')
```

Comment:
total nit, but it probably makes sense to move this up to the private symbols above since it's no longer "used by Reify mixin"

Reply:
Ah, good point.